themes/doom-ayu-mirage-theme.el

```diff
@@ -157,6 +157,8 @@ determine the exact padding."
     :inherit 'font-lock-comment-face
     :foreground doc-comments)
 
+   (cursor :background "DarkOrange1")
```

Comment:
Please use one of our color variables, or use `doom-darken` or `doom-lighten` to adjust them.